### PR TITLE
WT-12443 Add stats to histograms in wt_verify.py

### PR DESCRIPTION
**Pull request by adelinexchen**

### CODE REVIEW

File: tools/wt_verify/wt_verify.py [hunk 1]
```diff
@@ -171,6 +171,59 @@ def parse_output():
...
 
+def mean(array):
+    return round(sum(array) / len(array), 2)
+
+
+def median(array):
+    return array[len(array) // 2]
+
+
+def num_len(num):
```
Comment (Sean04) [inline tools/wt_verify/wt_verify.py:182]: What is this function used for?

**Comment by marcbutler** [comment]
@Siddhartha8899 @Sean04 
Should this be re-reviewed and merged or closed?

**Comment by Siddhartha8899** [comment]
Perhaps we could leave it as is? I will get to it soon.

**Comment by marcbutler** [comment]
Sure. Do you have sense of when that might be?

**Comment by marcbutler** [comment]
@Siddhartha8899 I'd like to get this off my backlog. How can I help advance this to resolution?

File: tools/wt_verify/wt_verify.py [hunk 2]
```diff
@@ -184,6 +185,21 @@ def histogram(field, chkpt, chkpt_name):
             elif "leaf" in metadata["page_type"]:
                 leaf.append(metadata[field])
 
+    # Using panda to calculate stats.
```
Comment (Siddhartha8899) [inline tools/wt_verify/wt_verify.py:188]:
```suggestion
    # Using pandas to calculate stats.
```

**Review by etienneptl — APPROVED** [review]

File: tools/wt_verify/wt_verify.py [hunk 3]
```diff
@@ -184,6 +185,21 @@ def histogram(field, chkpt, chkpt_name):
             elif "leaf" in metadata["page_type"]:
                 leaf.append(metadata[field])
 
+    # Using pandas to calculate stats.
+    df_internal = pd.DataFrame(internal, columns=[field])
+    df_leaf = pd.DataFrame(leaf, columns=[field])
+
+    stats_internal = df_internal.describe().loc[['mean', '50%', 'min', 'max']].transpose()
```
Comment (etienneptl) [inline tools/wt_verify/wt_verify.py:192]: You could create variables for:

```
['mean', '50%', 'min', 'max']
['Mean', 'Median', 'Min', 'Max']
```

Comment (Siddhartha8899) [reply tools/wt_verify/wt_verify.py]: Updated in [94b7f36](https://github.com/wiredtiger/wiredtiger/pull/10230/commits/94b7f3678b2778502e99a117a45e702975c36107)

**Review by Sean04 — APPROVED** [review]
lgtm

**Pull request merged**